lib/transports/ws2.js

```diff
@@ -465,11 +465,9 @@ class WSv2 extends EventEmitter {
    * @private
    */
   _updateManagedOB (symbol, data) {
-    if (Array.isArray(data[0])) { // snapshot, new OB
-      if (this._orderBooks[symbol]) {
-        return new Error(`recv snapshot for known OB: ${symbol}`)
-      }
-
+    // Snapshot, new OB. Note that we don't protect against duplicates, as they
+    // could come in on re-sub
+    if (Array.isArray(data[0])) {
```

Comment:
its not clear to me what that fixes?

maybe write it int he commit message or test?

Reply:
hmmm... what was the protection against duplicates used for?

Reply:
It was a mistake I made initially, I assumed the snapshots only ever arrive once per-connection, not once per-subscription. With the old behavior an error would be thrown if a duplicate snapshot arrived (meant as a sanity check.)